testsuite/MDAnalysisTests/core/test_segment.py

```diff
@@ -69,12 +69,13 @@ def test_generated_residueselection():
     """Test that a generated residue group always returns a ResidueGroup (Issue 47)
     unless there is a single residue (Issue 363 change)"""
     universe = mda.Universe(PSF, DCD)
-    # only a single Cys in AdK
-    cys = universe.s4AKE.CYS
-    assert isinstance(cys, mda.core.groups.Residue), \
-        "Single Cys77 is NOT returned as a single Residue (Issue 47)"
+    with pytest.warns(DeprecationWarning):
```

Comment:
I think this behavior will be removed in 1.0. right? Add a comment that this test will change??